### PR TITLE
Docs: Reworded sentences to improve clarity

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
- Reworded and changed tense in a sentence.
- Added clarity by rearranging words.

### CODE REVIEW

**Comment by LakshmiLavanyaKasturi** [comment]
Relates to https://github.com/hirosystems/docs/issues/120

File: README.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ npm create stacks
 
 ## Options
 
-Running the `npm create stacks` command launches in interactive mode by default.
+Run the `npm create stacks` command to launch interactive mode by default.
 Similar commands exist in other package managers.
 e.g., `yarn create stacks`
 
-The [`npm init`](https://docs.npmjs.com/cli/v8/commands/npm-init) and its alias `npm create` can be seen as an alias for `npm exec create-<...>`.
+The [`npm init`](https://docs.npmjs.com/cli/v8/commands/npm-init) and it's alias `npm create` can be seen as an alias for `npm exec create-<...>`.
```
Comment (janniks) [inline README.md]: This one should stay a possessive `its`, right?

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 ## Quick-Start a Stacks.js Project
```
Comment (sabbyanandan) [inline README.md]: Could we add an intro/overview paragraph to describe this repo's intent or objective? Explaining that first and getting into the usage steps next would be helpful.

IMHO, this project aims to help quickly bootstrap frontend applications with Stacks.js on top of the desired JS framework as the foundation.

**Review by sabbyanandan — CHANGES_REQUESTED** [review]

**Comment by LakshmiLavanyaKasturi** [comment]
@sabbyanandan Looks good?

File: README.md [hunk 3]
```diff
@@ -6,9 +6,11 @@
 - <img src="./misc/icons/vue.png" width="16" alt="Vue icon"> [Live Stacks Vue Starter Template](https://stacks-vue.vercel.app)
 - <img src="./misc/icons/angular.png" width="16" alt="Angular icon"> [Live Stacks Angular Starter Template](https://stacks-angular.vercel.app)
 
-## Quick-Start a Stacks.js Project
+## Quick-Start a Stacks.js project
 
-The following command will spawn an interactive CLI, which allows you to select your favorite framework to use with Stacks.js and get started building in an instant! ⚡️
+This project aims to help quickly bootstrap frontend applications with Stacks.js on top of the desired JS framework as the foundation.
```
Comment (sabbyanandan) [inline README.md]: Perhaps move this to the top and above the template bullets? So folks know right away what this project is about.

Comment (sabbyanandan) [inline README.md]:
```suggestion
This project aims to help quickly bootstrap frontend applications with Stacks.js on top of the following Javascript frameworks as the foundation.
```

**Review by sabbyanandan — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (sabbyanandan) [reply README.md]: This statement would be true if we move this description to the top.

Comment (sabbyanandan) [reply README.md:11]: Maybe also expand JS = Javascript

**Review by sabbyanandan — APPROVED** [review]
@LakshmiLavanyaKasturi: Thanks!

**Comment by LakshmiLavanyaKasturi** [comment]
@janniks: Can I get write permissions to [Stacks.js-starters](https://github.com/hirosystems/stacks.js-starters) repo?

**Comment by janniks** [comment]
> @janniks: Can I get write permissions to [Stacks.js-starters](https://github.com/hirosystems/stacks.js-starters) repo?

Done ✅

File: README.md [hunk 4]
```diff
@@ -1,14 +1,16 @@
 # Stacks.js Starters 🚀
 
+This project aims to help quickly bootstrap frontend applications with Stacks.js on top of the following Javascript frameworks as the foundation.
```
Comment (janniks) [inline README.md]:
```suggestion
This project aims to help quickly bootstrap frontend applications with Stacks.js on top of the following JavaScript frameworks as the foundation.
```

**Review by janniks — APPROVED** [review]

**Pull request merged**